kalite/distributed/static/js/distributed/exercises/views.js

```diff
@@ -431,6 +443,7 @@ window.ExerciseView = Backbone.View.extend({
             this.related_video_view.remove();
         }
         this.$("input").qtip("destroy", /* immediate */ true);
+        this.closed = true;
```

Comment:
This seems like something general purpose, that should either already be part of Backbone or we should add as part of our ancestor class, in our `close` method?